src/app/view/terms-and-conditions/terms-and-conditions.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="container py-4">
+  <div class="row justify-content-center">
+    <div class="col-md-10">
+      <h1 class="mb-4">Terms and Conditions</h1>
```

Comment:
can we just render markdown. It's easier that way since this document will continue to change ?